src/class.js

```diff
@@ -18,8 +25,23 @@
 // Cat should have the property `name` that is set in the constructor and the method
 // `meow` that should return the string `<name> meowed!` where `<name>` is the `name`
 // property set on the Cat instance.
-
-
+class Animal {
+	constructor(options) {
+		this.age = options.age;
+	}
+	growOlder() {
+		return this.age++;
```

Comment:
Something funny to be aware of here is that the `++` postfix operator will apply the incrementation AFTER the value has been returned.  So if you have `1` and you go `return 1++;` a `1` is returned and then it is turned into a `2`.